test/cli/package-autocorrect-missing-import/test.out

```diff
@@ -277,3 +277,62 @@ class Foo::MyPackage < PackageSpec
   strict_dependencies 'layered'
   test_import Foo::Bar::FalseAndAppPackage
 end
+
+--------------------------------------------------------------------------
+
+use_cycle_package/foo.rb:6: `Foo::Bar::CyclePackage::OtherClass` resolves but its package is not imported. However, it cannot be automatically imported because importing it would put `Foo::MyPackage` into a cycle https://srb.help/3718
```

Comment:
I worry that especially in places like VS Code, where there's not a ton of space, people might only see the first part and not the "however" part, which is arguably more important.

What do you think about, in these cases, making the error say something like

"Referencing `...::OtherClass` puts `Foo::MyPackage` into a dependency cycle"

and then maybe also say something in an error line like, "Foo::MyPackage does not import `Foo::Bar::CyclePackage::OtherClass`, so no cycle existed previously. Even if it were imported, there would still be a cycle."

not sold on any of these messages. I just think that the "cycle" aspect is the most important part of this message, and it's currently the very last word of what will shape up to be an incredibly long error header.

Reply:
secondary constraint: we try not to have `.` (periods, i.e. multiple sentences) in error headers, because it suggests that the header is too long or written with the wrong tone/style

Reply:
Ya, I agree. I think I may just get rid of the "resolves but its package is not imported" part entirely, it doesn't really matter in fixing this issue. I just kept it now in the message for consistency, will improve the error messages for all the cases together in a future PR.